tutorials/firefly/OpenUniverse2024Preview_Firefly.md

```diff
@@ -0,0 +1,750 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using Firefly to Explore OpenUniverse2024 Data Preview Simulated Roman and Rubin Images
+
++++
+
+## Learning Goals
+
+By the end of this tutorial, you will:
+
+- Learn how to access cloud-hosted Roman and Rubin simulated images.
+
+- Learn how to launch an interactive Firefly instance inside JupyterLab.
+
+- Learn how to use the Firefly Jupyterlab extension to visualize cloud-hosted simulated images, overplot ds9 regions, overplot catalogs in Parquet format, and create 3 color images.
+
++++
+
+## Introduction
+
+The purpose of this tutorial is to become familiar with simulated Roman and Rubin simulated data published through the OpenUniverse 2024 data preview, and to become familiar with the Firefly JupyterLab Extension for visualizing astronomical data products.
+
+OpenUniverse2024 is a project to simulate spatially overlapping imaging surveys to be carried out by the Nancy Grace Roman Telescope and the Vera C. Rubin Observatory. The simulations were carried out on Argonne's Theta cluster and consist of:
+
+- The LSST ELAIS-S1 Deep Drilling Field (DDF)
+- The Roman Time-Domain Survey (TDS) shifted to overlap the ELAIS region and LSST DDF
+- Overlapping LSST Wide-Fast-Deep (WFD) survey (with rolling cadence)
+- Overlapping Roman Wide-Area Survey (WAS) in the same region
+- A deep-field calibration region of the Roman WAS in the same region
+
+This data preview release consists of a subset of data from each of the five categories above. More information about data preview can be found at:
+https://irsa.ipac.caltech.edu/data/theory/openuniverse2024/overview.html
+
+Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed at Caltech and used by multiple space- and ground-based astrophysics archives. More information can be found here:
+https://github.com/Caltech-IPAC/firefly/blob/dev/README.md
+
+In addition to being used to make web applications, Firefly can be used from Python. More information can be found here:
+https://caltech-ipac.github.io/firefly_client/usage/index.html
+
+The Firefly JupyterLab Extension makes it particularly easy to use Firefly to efficiently visualize cloud-hosted astronomical data using cloud-based JupyterLab instances. More information can be found here:
+
+https://github.com/Caltech-IPAC/jupyter_firefly_extensions/blob/master/README.md
+
++++
+
+## Imports
```

Comment:
I don't think we need to go into this much details for each submodules, but since these are here I don't advocate for their compression, only if the scientists would ask for it, too.

Reply:
Yes, I'll defer it to Vandana